### PR TITLE
[fix] multilingual duckduckgo

### PR DESCRIPTION
**Pull request by MarcAbonce**
Language support for duckduckgo engine.

### CODE REVIEW

File: searx/engines/duckduckgo.py [hunk 1]
```diff
@@ -39,13 +37,22 @@ def request(query, params):
...
-
-    params['url'] = url.format(
-        query=urlencode({'q': query, 'kl': locale}),
-        offset=offset)
+        locale = params['language'].split('_')
+        if len(locale) == 2:
+            # country code goes first
+            locale = locale[1].lower() + '-' + locale[0].lower()
+        else:
+            # doesn't actually do anything because ddg requires both country and language
```
Comment (kvch) [inline searx/engines/duckduckgo.py]: I tried your PR with a locale that is only one long. It did not work as you had written here. If DDG requires both country and language why did you include the `else`? Is it possible that there will be locales which match the format DDG requires without any transformation?

**Comment by kvch** [comment]
Apart from the question I added inline, I think it is cool. :)

**Comment by MarcAbonce** [comment]
You're right. I just changed it to try to guess the country if none is given.
So language support should work now, even if no country is specified by the user.

**Comment by asciimoo** [comment]
nice, thanks

**Pull request merged**